### PR TITLE
added functionality - update product quantity, when it's was purchased

### PR DESCRIPTION
**Pull request by SergiuLA17**

### CODE REVIEW

File: src/main/resources/db/migration/V2__add_data.sql [hunk 1]
```diff
@@ -1,4 +1,11 @@
 INSERT INTO `Storehouse`.`products` (`name`, `quantity`, `date_of_manufacture`, `days_to_expire`)VALUES ('Salt', '1000', LOCALTIME, '3000');
 INSERT INTO `Storehouse`.`products` (`name`, `quantity`, `date_of_manufacture`, `days_to_expire`)VALUES ('Sugar', '3000', LOCALTIME, '5000');
 INSERT INTO `Storehouse`.`products` (`name`, `quantity`, `date_of_manufacture`, `days_to_expire`)VALUES ('Potato', '500', LOCALTIME, '50');
-INSERT INTO `Storehouse`.`products` (`name`, `quantity`, `date_of_manufacture`, `days_to_expire`)VALUES ('Aplle', '300', LOCALTIME, '30');
\ No newline at end of file
+INSERT INTO `Storehouse`.`products` (`name`, `quantity`, `date_of_manufacture`, `days_to_expire`)VALUES ('Apple', '700', LOCALTIME, '15');
```
Comment (arthurdobrea) [inline src/main/resources/db/migration/V2__add_data.sql:4]: Thast should be added in separate file. V3_add_data for example. Every change that you are making in database, even if its 1 row of sql code, should be done in separate file.

File: src/main/java/com/example/Storehouse/service/StorehouseService.java [hunk 2]
```diff
@@ -1,17 +1,48 @@
...
+        Products products = findOne(product.getIdProduct());
+        products.setQuantity(200 + (int) (Math.random() * 500));
+        logger.info("Product <"+products.getNameOfProduct() + "> is over, loading more. Now in store is " + products.getQuantity() + " kg");
+
+        productRepository.save(products);
+    }
+
+
+    public Products findOne(int id) {
+        return productRepository.findById(id).orElse(null);
```
Comment (arthurdobrea) [inline src/main/java/com/example/Storehouse/service/StorehouseService.java]: Never return null, you could get null pointer exception in row 27, think how you can rewrite this method.
You can use Optional class, or better google how it should be done.

File: src/main/resources/db/migration/V2__add_data.sql (hunk 1 above)
Comment (SergiuLA17) [reply src/main/resources/db/migration/V2__add_data.sql:4]: done

File: src/main/java/com/example/Storehouse/service/StorehouseService.java (hunk 2 above)
Comment (SergiuLA17) [reply src/main/java/com/example/Storehouse/service/StorehouseService.java]: done

**Review by arthurdobrea — APPROVED** [review]

**Pull request merged**